### PR TITLE
OpenCV 4.5.0

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

**Comment by extrowerk** [comment]
Patch upstreamed, will be available in the next revision.

**Comment by Begasus** [comment]
Anything I could do to check this out on 32 bit @extrowerk ?

File: media-libs/opencv/opencv4-4.4.0.recipe [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+#	devel:libtesseract$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libwebp$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+#	cmd:ant
+	cmd:cmake
+	cmd:doxygen
+	cmd:f2py2
```
Comment (Begasus) [inline media-libs/opencv/opencv4-4.4.0.recipe:168]: Recipe for numpy_python-1.16.5.recipe is disabled for secondary architecture, thus only available at 64 bit (or with a local build

File: media-libs/opencv/opencv4-4.4.0.recipe [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+	lib:libgstreamer_1.0$secondaryArchSuffix
+	lib:libgstvideo_1.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libhdf5$secondaryArchSuffix
+#	lib:libilmimf$secondaryArchSuffix #openexr
+	lib:libintl$secondaryArchSuffix
+	lib:libjasper$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:liblapack$secondaryArchSuffix
+	lib:libopenblas$secondaryArchSuffix
```
Comment (Begasus) [inline media-libs/opencv/opencv4-4.4.0.recipe:73]: For some reason missing for secondary architecture build (32 bit)?
```diff
+	lib:libopenjp2$secondaryArchSuffix
```
Added them in the recipe (lib + devel section)

**Review by Begasus — COMMENTED** [review]
Build hasn't finished yet, but looking good so far, just wanted to drop my 2 cents here :)

**Comment by Begasus** [comment]
Got a working build, but install part fails in many ways, did you check previous version?

**Comment by extrowerk** [comment]
Will do, thanks, great job!

**Comment by extrowerk** [comment]
I think it should be ok now.

**Pull request merged**